fields/types/datetime/DatetimeType.js

```diff
@@ -84,16 +81,22 @@ datetime.prototype.inputIsValid = function (data, required, item) {
  * Updates the value for this field in the item from a data object
  */
 datetime.prototype.updateItem = function (item, data, callback) {
-	var input = this.getInputFromData(data);
-	if (input === undefined) return process.nextTick(callback);
-	var newValue = this.parse(input);
-	var oldValue = item.get(this.path);
-	if (newValue.isValid()) {
-		if (!oldValue || !newValue.isSame(oldValue)) {
-			item.set(this.path, newValue.toDate());
+	// Get the values from the data
+	var value = this.getInputFromData(data);
+	if (value !== undefined) {
+		if (value !== null && value !== '') {
+			// If the value is not null, empty string or undefined, parse it
+			var newValue = this.parse(value, this.formatString);
+			// If it's valid and not the same as the last value, save it
+			if (newValue.isValid()) {
```

Comment:
This should possibly not be in the updateItem, since the validation should already be done by the point it reaches this in `validateInput`/`validateRequiredInput`?